projects/ziti-console-lib/src/lib/features/wrappers/zac-wrapper.service.ts

```diff
@@ -812,6 +813,26 @@ export class ZacWrapperService extends ZacWrapperServiceClass {
         });
     }
 
+    redefineObject(obj) {
+        for (let prop in obj) {
+            if (Array.isArray(obj[prop]) && obj[prop].length==0) {
+                delete obj[prop];
+            } else {
+                if (typeof obj[prop] === "string" && obj[prop].trim().length==0) {
+                    delete obj[prop];
+                } else {
+                    if (typeof obj[prop] === "object") {
+                        obj[prop] = this.redefineObject(obj[prop]);
+                        if (Object.keys(obj[prop]).length==0) {
+                            delete obj[prop];
+                        }
+                    }
```

Comment:
there's no 'else' here. feels like a console.log or ... anything ? would be good here?

Reply:
are numbers covered? or just left the same?

Reply:
> are numbers covered? or just left the same?

Should be left the same